### PR TITLE
Add Translating AMR values instructions

### PR DESCRIPTION
**Pull request by sahandilshan**
## Purpose
* $subject

## Related PRs
* https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1814
* https://github.com/wso2/carbon-identity-framework/pull/3954

## Related Issue
* https://github.com/wso2/product-is/issues/13225

### CODE REVIEW

File: en/docs/learn/working-with-acr-and-amr.md [hunk 1]
```diff
@@ -274,6 +274,49 @@ Follow the steps below to try the scenario without using an application:
 	
 	Note that the AMR values in this example are 	`DemoFaceIdAuthenticator`	, 	`BasicAuthenticator`	, and 	`DemoFingerprintAuthenticator`. These are the authenticators that are used in the authentication process.
 
+#### Translating AMR values
+Follow the steps below to translate the AMR values that get returned to the authentication process.
```
Comment (nilminiwso2) [inline en/docs/learn/working-with-acr-and-amr.md:278]:
```suggestion
Follow the steps below to translate the AMR values that get returned through the authentication process.
```

File: en/docs/learn/working-with-acr-and-amr.md [hunk 2]
```diff
@@ -274,6 +274,49 @@ Follow the steps below to try the scenario without using an application:
...
+
+1. Open the `deployment.toml` file in the `<IS_HOME>/repository/conf directory`.
+
+2. Add the following configurations
```
Comment (nilminiwso2) [inline en/docs/learn/working-with-acr-and-amr.md:282]:
```suggestion
2. Add the following configurations:
```

File: en/docs/learn/working-with-acr-and-amr.md [hunk 3]
```diff
@@ -274,6 +274,49 @@ Follow the steps below to try the scenario without using an application:
...
+
+    !!! note 
+    You can add any custom AMR mapping to this config.
```
Comment (nilminiwso2) [inline en/docs/learn/working-with-acr-and-amr.md:285]: Does this render properly when you build the doc locally?
I believe it should be aligned as:

!!! note
    You can add

File: en/docs/learn/working-with-acr-and-amr.md [hunk 4]
```diff
@@ -274,6 +274,49 @@ Follow the steps below to try the scenario without using an application:
...
+       method = "SMSOTP"
+       amr_value = "sms"
+       [[authentication_context.method_refs]]
+       method = "EmailOTP"
+       amr_value = "otp"
+       [[authentication_context.method_refs]]
+       excluded_methods = ["BasicAuthenticator", "AuthenticationMethodToBeIgnored"]
+       ```
+	
+    `method`: Authentication Method Name
```
Comment (nilminiwso2) [inline en/docs/learn/working-with-acr-and-amr.md:298]: In bullet points:

- `method`: Name of the authentication method.
- `amr_value`: The AMR value mapping you prefer to have as the AMR in the ID token. See [AMR draft specification](https://tools.ietf.org/html/draft-ietf-oauth-amr-values-04) for details.
- `excluded_methods`: The list of authentication methods that you would like to exclude from the ID token.

File: en/docs/learn/working-with-acr-and-amr.md [hunk 5]
```diff
@@ -274,6 +274,49 @@ Follow the steps below to try the scenario without using an application:
...
+
+    `amr_value`: AMR value mapping you prefer to have as the amr in the ID token. You can refer [AMR draft specification](https://tools.ietf.org/html/draft-ietf-oauth-amr-values-04) for this.
+
+	`excluded_methods`: List of Authentication Methods that you would like to prevent being available on the ID token.
+
+4. Start the identity server
```
Comment (nilminiwso2) [inline en/docs/learn/working-with-acr-and-amr.md:304]:
```suggestion
4. Start the WSO2 Identity Server.
```

File: en/docs/learn/working-with-acr-and-amr.md [hunk 6]
```diff
@@ -274,6 +274,49 @@ Follow the steps below to try the scenario without using an application:
...
+
+!!! info
+You can verify the whether the changes have been successfully applied by opening the `identity.xml` file in the `<IS_HOME>/repository/conf/identity` directory after starting the server. It should contain a similar configuration as follows,
```
Comment (nilminiwso2) [inline en/docs/learn/working-with-acr-and-amr.md:307]:
```suggestion
You can verify whether the changes are successfully applied by opening the `identity.xml` file in the `<IS_HOME>/repository/conf/identity` directory after starting the server. It should contain a similar configuration as shown below.
```

File: en/docs/learn/working-with-acr-and-amr.md [hunk 7]
```diff
@@ -274,6 +274,49 @@ Follow the steps below to try the scenario without using an application:
...
+	
+    `method`: Authentication Method Name
+
+    `amr_value`: AMR value mapping you prefer to have as the amr in the ID token. You can refer [AMR draft specification](https://tools.ietf.org/html/draft-ietf-oauth-amr-values-04) for this.
+
+	`excluded_methods`: List of Authentication Methods that you would like to prevent being available on the ID token.
+
+4. Start the identity server
+
+!!! info
```
Comment (nilminiwso2) [inline en/docs/learn/working-with-acr-and-amr.md:306]: Please double check if this renders properly

File: en/docs/learn/working-with-acr-and-amr.md (hunk 4 above)
Comment (sahandilshan) [reply en/docs/learn/working-with-acr-and-amr.md]: Fix with a5c790700af9084435520e134ba8f71368868655

File: en/docs/learn/working-with-acr-and-amr.md (hunk 3 above)
Comment (sahandilshan) [reply en/docs/learn/working-with-acr-and-amr.md]: Fix with a5c790700af9084435520e134ba8f71368868655
Screenshot 2022-04-28 at 9 49 48 PM

File: en/docs/learn/working-with-acr-and-amr.md (hunk 7 above)
Comment (sahandilshan) [reply en/docs/learn/working-with-acr-and-amr.md:306]: Fix with a5c790700af9084435520e134ba8f71368868655
Screenshot 2022-04-28 at 9 49 56 PM

File: en/docs/learn/working-with-acr-and-amr.md [hunk 8]
```diff
@@ -274,6 +274,49 @@ Follow the steps below to try the scenario without using an application:
 	
 	Note that the AMR values in this example are 	`DemoFaceIdAuthenticator`	, 	`BasicAuthenticator`	, and 	`DemoFingerprintAuthenticator`. These are the authenticators that are used in the authentication process.
 
+#### Translating AMR values
+Follow the steps below to translate the AMR values that get returned through the authentication process.
+
+1. Open the `deployment.toml` file in the `<IS_HOME>/repository/conf directory`.
```
Comment (nilminiwso2) [inline en/docs/learn/working-with-acr-and-amr.md:280]:
```suggestion
1. Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory.
```

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**